activiti-core-common/activiti-spring-cache-manager/src/test/java/org/activiti/spring/cache/test/ActivitiSpringNoopCacheManagerTests.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2010-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.spring.cache.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.cache.CacheType;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.support.NoOpCache;
+import org.springframework.cache.support.NoOpCacheManager;
+
+@SpringBootTest(
+    properties = {
+        "debug=false",
+        "activiti.spring.cache-manager.provider=noop",
+        "activiti.spring.cache-manager.caches.foo.enabled=true",
+        "activiti.spring.cache-manager.caches.bar.enabled=false"
+})
+public class ActivitiSpringNoopCacheManagerTests {
```

Comment:
Tests or Test? 🤔
Could you remove publics that are no longer needed with JUnit 5?